app/models/owner_statistic.rb

```diff
@@ -40,50 +40,50 @@ def collection_ids
     [self.all_owner_collections.ids]
   end
 
-  def subject_count(last_days=nil)
-    owner_subjects.where("#{last_days_clause(last_days, 'created_on')}").count
+  def subject_count(days=nil)
+    owner_subjects.where(date_range_clause(days, 'created_on')).count
   end
 
-  def mention_count(last_days=nil)
-    PageArticleLink.where(article_id: owner_subjects.ids).where("#{last_days_clause(last_days, 'created_on')}").count
+  def mention_count(days=nil)
+    PageArticleLink.where(article_id: owner_subjects.ids).where(date_range_clause(days, 'created_on')).count
   end
 
-  def contributor_count(last_days=nil)
-    User.joins(:deeds).where(deeds: {collection_id: collection_ids}).where("#{last_days_clause(last_days, 'deeds.created_at')}").distinct.count
+  def contributor_count(days=nil)
+    User.joins(:deeds).where(deeds: {collection_id: collection_ids}).where(date_range_clause(days, 'deeds.created_at')).distinct.count
```

Comment:
We've just heard from the client that they only want collaborators who actually worked on the project.  Since we have a "joined collection" deed that doesn't represent any actual activity, we need to filter by deed_type to make sure it's not included.